tests/quantization/bnb/test_mixed_int8.py

```diff
@@ -914,14 +915,9 @@ def test_training(self):
         batch = self.tokenizer("Test batch ", return_tensors="pt").to(torch_device)
 
         # Step 4: Check if the gradient is not None
-        if torch_device in {"xpu", "cpu"}:
-            # XPU and CPU finetune do not support autocast for now.
+        with torch.autocast(torch_device):
```

Comment:
Is there a minimum pytorch/ipex version required for this?

Reply:
No restriction for pytorch/ipex, this was fixed in BNB.